src/hypertable.c

```diff
@@ -2911,4 +2936,4 @@ ts_hypertable_update_dimension_partitions(const Hypertable *ht)
 	}
 
 	return false;
-}
```

Comment:
You seem to have accidentally removed the newline at the end of the file.

Reply:
I do not see any change in this file at the end.

Reply:
There is a "red sign" below this comment. If you hover over it, it shows that there is an newline missing.